server/server.go

```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/0xPolygon/polygon-edge/blockchain/storagev2/memory"
 	"github.com/0xPolygon/polygon-edge/chain"
 	"github.com/0xPolygon/polygon-edge/consensus"
-	consensusPolyBFT "github.com/0xPolygon/polygon-edge/consensus/polybft"
+	consensusPolyBFT "github.com/0xPolygon/polygon-edge/consensus/polybft/config"
```

Comment:
```suggestion
	polycfg "github.com/0xPolygon/polygon-edge/consensus/polybft/config"
```
In order to use the same package name across all files.

Reply:
Fix: https://github.com/Ethernal-Tech/blade/pull/383/commits/a427654ff06d6d739b396e80302b09b8c50d39d0